devel/re-flex/Portfile

```diff
@@ -26,8 +26,9 @@ long_description    RE/flex is a regex-centric, fast lexical analyzer generator
 
 homepage            https://www.genivia.com/doc/reflex/html/
 
-# identical executable name
-conflicts           reflex
+notes               "The executable and manpage are renamed to avoid conflict with the pre-existing \"reflex\" port."
```

Comment:
perhaps mention to *what* they are renamed?

Reply:
> perhaps mention to _what_ they are renamed?

done:
```
$ sudo port notes re-flex
--->  re-flex has the following notes:
  The executable and manpage are renamed to "re-flex" (consistent with the port
  name), avoiding conflict with the pre-existing "reflex" port.
```